v3/testdata/policyQualifiersOtherThanCpsNotPermittedError.pem

```diff
@@ -0,0 +1,85 @@
+Certificate:
+    Data:
+        Version: 3 (0x2)
+        Serial Number: 18008675309 (0x4316693ed)
+        Signature Algorithm: sha256WithRSAEncryption
+        Issuer: C = US, O = Mother Nature, OU = Everything, CN = Mother Nature
+        Validity
+            Not Before: Dec  1 06:07:08 2055 GMT
+            Not After : Aug 21 01:33:21 2056 GMT
+        Subject: C = US, O = Extreme Discord, OU = Chaos, L = Tallahassee, ST = FL, street = 3210 Holly Mill Run, postalCode = 30062, CN = gov.us
+        Subject Public Key Info:
+            Public Key Algorithm: rsaEncryption
+                Public-Key: (512 bit)
+                Modulus:
+                    00:e4:91:22:90:c9:c9:2e:d7:ae:02:08:23:71:57:
+                    60:6e:0c:31:46:5d:49:01:d8:ce:b6:42:06:58:46:
+                    3f:6c:2d:da:65:45:06:95:94:5d:0d:3d:ee:33:70:
+                    76:ad:4b:dc:18:54:1e:5d:92:60:c4:79:e3:fe:79:
+                    4d:4d:f1:18:8f
+                Exponent: 65537 (0x10001)
+        Issuer Unique ID:             01:02:03
+        X509v3 extensions:
+            X509v3 Key Usage: critical
+                Digital Signature, Key Encipherment, Certificate Sign
+            X509v3 Extended Key Usage: 
+                TLS Web Client Authentication, TLS Web Server Authentication
+            X509v3 Basic Constraints: critical
+                CA:TRUE
+            X509v3 Authority Key Identifier: 
+                01:02:03
+            X509v3 Certificate Policies: 
+                0..F0.. ..g.....0...0...+.......0.0.....+.......0..0
..0.............An explicitText field includes the textual statement directly in the certificate.  The explicitText field is a string with a maximum size of 200 characters.  Conforming CAs SHOULD use the UTF8String encoding for explicitText.  0.....*...0...0...+.......0.0.....+.......0..0
..0.............An explicitText field includes the textual statement directly in the certificate.  The explicitText field is a string with a maximum size of 200 characters.  Conforming CAs SHOULD use the UTF8String encoding for explicitText.  
```

Comment:
Clever use of the test certificate to also document what the certificate is testing :wink:

Reply:
Noted for my next time!